### ISSUE TITLE
Feature request: exclude certain 'notebooks' files from firing reminders

### ISSUE THREAD

**Issue by srustamo** [orgzly/orgzly-android#245]
Currently all 'notebooks' which have relevant timestamps fire reminders.

I currently use `org-mobile` package in Emacs, with exports of relevant 'notebooks' set to Dropbox. It also includes its own 'agenda' notebook. So, a scheduled item appears in Orgzly in both in the source 'notebook' and also in 'agenda' notebook generated by `org-mobile-push` in Emacs.

**Comment by nevenz** [comment]
Search query (or queries) could be made to control this.

**Comment by srustamo** [comment]
Can you please elaborate? Do you mean that reminders can be selective?

**Comment by nevenz** [comment]
No, I was just throwing the idea for future implementation. Allowing to use `.b.agenda or t.silenced` for this.

**Comment by srustamo** [comment]
Sounds good.

**Comment by karlicoss** [comment]
Yeah, using a tag query for reminders would be awesome! I've got quite a few tasks which have scheduled date, but that not worth showing a reminder on mobile. 
@nevenz you know if anyone started progress on that? If not I might give it a try.

**Comment by nevenz** [comment]
> you know if anyone started progress on that? If not I might give it a try.

I'm not aware of any. That would be great, thanks.

I guess it would be a new preference in Settings / Reminders that can contain a search query?
